checkout/rust/delivery-customization/default/src/main.rs

```diff
@@ -0,0 +1,40 @@
+use serde::Serialize;
+
+mod api;
+use api::*;
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let input: Input = serde_json::from_reader(std::io::BufReader::new(std::io::stdin()))?;
+    let mut out = std::io::stdout();
+    let mut serializer = serde_json::Serializer::new(&mut out);
+    function(input)?.serialize(&mut serializer)?;
+    Ok(())
+}
+
+fn function(_input: Input) -> Result<FunctionResult, Box<dyn std::error::Error>> {
+    Ok(FunctionResult { operations: vec![] })
```

Comment:
Does it make sense to make it always hide the first delivery option or something? Just so we know that it's running

Reply:
Resolved with Slack discussion, will follow suit with current implementation as this is how other `default` sample functions are designed.